### ISSUE TITLE
onDelete accepts any value, even invalid ones.

### ISSUE THREAD

**Issue by Celc** [propelorm/Propel2#870]
According to the spec:

    onDelete = "cascade|setnull|restrict|none"

**Comment by marcj** [comment]
Yes, is actually a good idea to throw an exception, although we have already .xsd files which makes it possible to highlight errors in your IDE.
